sesman/chansrv/chansrv.c

```diff
@@ -1791,6 +1792,13 @@ main(int argc, char **argv)
         return 1;
     }
 
+    restrict_outbound_clipboard_env = g_getenv("CHANSRV_RESTRICT_OUTBOUND_CLIPBOARD");
+    if (restrict_outbound_clipboard_env != 0) {
```

Comment:
See `coding_style.md`.  I know some existing codes don't obey our coding style but newly added codes should obey the coding style.